members/models/activityparticipant.py

```diff
@@ -86,8 +85,10 @@ def payment_info(self, format_as_html: bool):
                 if payment.activity.price_in_dkk == 0:
                     result_string = f"{html_good_pre}Gratis.{html_post} "
                 else:
-
-                    if self.activity.start_date.year > timezone.now().year:
+                    if (
+                        payment.accepted_at is not None
+                        and self.activity.start_date.year > timezone.now().year
+                    ):
                         result_string = f"{html_warn_pre}Betalingsdato:{str(self.activity.start_date.year)}-01-01{html_post} "

```

Comment:
Ikke direkte relateret til dette PR, men er betalingsdato altid 1. januar?

Reply:
lige nu er logikken at det er 1. januar ..

Reply:
såfremt at aktiviteten er "næste år" eller senere..

Reply:
Havde det været på mit arbejde ville jeg nok have sagt at det skulle have været en eller anden form for variabel eller noget, men i dette tilfælde er det fint nok.